plugins/hls-explicit-imports-plugin/test/testdata/OnlyThis.expected.hs

```diff
@@ -1,7 +0,0 @@
-module OnlyThis where
```

Comment:
Not removed, just renamed. Previously there were name clashes (because most of refine imports was just copied from explicit imports), so I prefixed the specific action before the testdata files.